docs/user/docs/projects/models/backups/setup.md

```diff
@@ -2,3 +2,44 @@
  ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
  ~ SPDX-License-Identifier: Apache-2.0
  -->
+
+# Setup a backup for a TeamForCapella model
+
+<!-- prettier-ignore -->
+!!! warning
+    You need to have the Administrator or Project Lead role for the specific
+    project to perform the following steps.
+
+<!-- prettier-ignore -->
+!!! danger
+    Models are not automatically backed up! Please make sure to set up a nightly backup for productive and important content.
+
+We offer the feature to create a backup from any TeamForCapella repository to any Git repository. These can be triggered manually or can be executed every night at around 3pm.
+
+<!-- prettier-ignore-start -->
+
+1. Navigate to `Projects`
+1. Select the relevant project
+1. Make sure that you have added at least one TeamForCapella model (source) and
+   at least one Git repository to the model (destination)
+1. In the model overview, select the `Synchronize`-button
+
+    ![Model overview](./model-overview.png)
+
+1. You can see a list of all registered backup pipelines.
+   In our case, there is not registered backup pipeline yet.
+
+    ![Backup pipeline](./backups-overview.png)
+
+1. Click on the `Plus`-icon on the top right and select the corresponding configuration.
+   You have to select a Git model (source) and a T4C model (destination).
+
+    ![Create a backup pipeline](./create-pipeline.png)
+
+    !!! Bug
```

Comment:
Is this really a bug? I think it's more like a warning.

The "Include commit history" does only deal with the new commits to the T4C repo or is it working it's way through the whole repo history?